src/cmd/build.js

```diff
@@ -1,30 +1,31 @@
 /* @flow */
 import path from 'path';
-import minimatch from 'minimatch';
 import {createWriteStream} from 'fs';
-import streamToPromise from 'stream-to-promise';
+
+import minimatch from 'minimatch';
 import {fs} from 'mz';
+import streamToPromise from 'stream-to-promise';
 import parseJSON from 'parse-json';
 
 import defaultSourceWatcher from '../watcher';
 import {zipDir} from '../util/zip-dir';
 import getValidatedManifest, {getManifestId} from '../util/manifest';
 import {prepareArtifactsDir} from '../util/artifacts';
 import {createLogger} from '../util/logger';
-import {UsageError} from '../errors';
+import {UsageError} from '../errors'; //eslint-disable-line import/order
```

Comment:
It's because all imports need to be grouped together. It's not the most helpful message but it's telling you that you forgot to group some imports. Here is a patch that fixes it:

````diff
diff --git a/src/cmd/build.js b/src/cmd/build.js
index 17a71f9..18c4601 100644
--- a/src/cmd/build.js
+++ b/src/cmd/build.js
@@ -12,17 +12,13 @@ import {zipDir} from '../util/zip-dir';
 import getValidatedManifest, {getManifestId} from '../util/manifest';
 import {prepareArtifactsDir} from '../util/artifacts';
 import {createLogger} from '../util/logger';
-import {UsageError} from '../errors'; //eslint-disable-line import/order
+import {UsageError} from '../errors';
+import type {OnSourceChangeFn} from '../watcher';
+import type {ExtensionManifest} from '../util/manifest';
 
 
 const log = createLogger(__filename);
 
-/* eslint-disable import/order, import/imports-first */
-// Import flow types.
-import type {OnSourceChangeFn} from '../watcher';
-import type {ExtensionManifest} from '../util/manifest';
-/* eslint-disable import/order, import/imports-first */
-
 export function safeFileName(name: string): string {
   return name.toLowerCase().replace(/[^a-z0-9\.-]+/g, '_');
 }
````